### PR TITLE
Test covereage query database

### PR DESCRIPTION
**Pull request by larnaa**

### CODE REVIEW

File: bluecoins.sql [hunk 1]
```diff
@@ -6,7 +6,7 @@ CREATE TABLE ITEMTABLE(
     itemAutoFillVisibility INTEGER
 );
 
-CREATE TABLE sqlite_sequence(name, seq);
+CREATE TABLE sqlite_sequence(name, seq);sqlite3 connection commit
```
Comment (droserasprout) [inline bluecoins.sql:9]: Typo?

File: src/cluecoins/cli.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def _convert(base_currency: str, db_path: str) -> None:
     conn = open_copy(db_path)
 
     storage = Storage()
-    storage.init()
+    storage.create_table_quote()
```
Comment (droserasprout) [inline src/cluecoins/cli.py:73]: Maybe `create_quote_table`?

File: src/cluecoins/database.py [hunk 3]
```diff
@@ -52,12 +52,14 @@ def update_transaction(conn: Connection, id_: int, rate: Decimal, amount: Decima
...
-def iter_accounts(conn: Connection) -> Iterator[tuple[int, str, Decimal]]:
+def iter_accounts(
+    conn: Connection, old_currency: str = 'USDT', new_currency: str = 'USD'
+) -> Iterator[tuple[int, str, Decimal]]:
     for row in conn.cursor().execute(
         'SELECT accountsTableID, accountCurrency, accountConversionRateNew FROM ACCOUNTSTABLE;'
     ):
         id_, currency, rate = row
-        currency = currency.replace('USDT', 'USD')
+        currency = currency.replace(old_currency, new_currency)
```
Comment (droserasprout) [inline src/cluecoins/database.py:62]: I don't think we need to perform substitution in `iter_accounts` method itself. A separate method to replace by subs.table?

File: src/cluecoins/storage.py [hunk 4]
```diff
@@ -15,12 +15,12 @@ class Storage:
     """Create and managing the local SQLite database."""
 
     def __init__(self) -> None:
-        self._path = Path(xdg.xdg_data_home()) / 'bluecoins-cli' / 'bluecoins-cli.db'
+        self._path = Path(xdg.xdg_data_home()) / 'cluecoins' / 'cluecoins.db'
         self._path.parent.mkdir(parents=True, exist_ok=True)
         self._path.touch(exist_ok=True)
         self._db = connect(self._path)
```
Comment (droserasprout) [inline src/cluecoins/storage.py:21]: Side effect. Fix or mark with FIXME

**Review by droserasprout — APPROVED** [review]

File: src/cluecoins/database.py (hunk 3 above)
Comment (larnaa) [reply src/cluecoins/database.py:62]: Task #36

**Pull request merged**